resources/js/Components/Form/CharacterCreateForm.test.js

```diff
@@ -17,10 +17,24 @@ const editProps = {
         status: 'private',
     },
     editId: 0,
+    factions: [
```

Comment:
for coding standard, we should always make a constant when the code is repeated twice, in this case you should make a constant for factions but since this is a test file, i dont really care

Reply:
Thanks, you're absolutely right! I've changed the test to use a single constant for factions.